pom.xml

```diff
@@ -56,7 +56,7 @@
       <dependency>
         <groupId>com.google.guava</groupId>
         <artifactId>guava</artifactId>
-        <version>20.0</version>
+        <version>24.1.1-jre</version>
```

Comment:
Doesn't this also need a version update for this pom itself?

Reply:
Nevermind, this is a SNAPSHOT. So, don't need to bump up the version.

Reply:
The versions are automatically updated with maven release plugin.